common/stream-model.cpp

```diff
@@ -779,177 +773,229 @@ namespace rs2
+
+                ImGui::PushStyleColor( ImGuiCol_Text, white );
+                ImGui::Text( "%s", text.c_str() );
+                ImGui::PopStyleColor();
+                
+                if( at.description != "" )
+                {
+                    if( ImGui::IsItemHovered() )
                     {
-                        std::string text = "";
-                        if (at.name != "") text = rsutils::string::from() << at.name << ":";
-                        auto size = ImGui::CalcTextSize(text.c_str());
-
-                        for (int i = 3; i > 0; i -= 1)
-                            ImGui::GetWindowDrawList()->AddRectFilled({ curr_info_rect.x + 10 - i, line_y - i },
-                                { curr_info_rect.x + 10 + i + size.x, line_y + size.y + i },
-                                ImColor(alpha(sensor_bg, 0.1f)));
-
-                        ImGui::PushStyleColor(ImGuiCol_Text, white);
-                        ImGui::Text("%s", text.c_str()); ImGui::SameLine();
-
-                        if (at.description != "")
-                        {
-                            if (ImGui::IsItemHovered())
-                            {
-                                ImGui::SetTooltip("%s", at.description.c_str());
-                            }
-                        }
-
-                        text = at.value;
-                        size = ImGui::CalcTextSize(text.c_str());
-
-                        for (int i = 3; i > 0; i -= 1)
-                            ImGui::GetWindowDrawList()->AddRectFilled({ curr_info_rect.x + 20 + max_text_width - i, line_y - i },
-                                { curr_info_rect.x + 30 + max_text_width + i + size.x, line_y + size.y + i },
-                                ImColor(alpha(sensor_bg, 0.1f)));
-
-                        ImGui::PopStyleColor();
-
-                        ImGui::SetCursorScreenPos({ curr_info_rect.x + 20 + max_text_width, line_y });
+                        ImGui::SetTooltip( "%s", at.description.c_str() );
+                    }
+                }
 
-                        std::string id = rsutils::string::from() << "##" << at.name << "-" << profile.unique_id();
+                text = at.value;
+                size = ImGui::CalcTextSize(text.c_str());
+                
+                // Draw a smooth rectangle background for each label value.
+                for (auto i = 3; i > 0; i--)
+                    ImGui::GetWindowDrawList()->AddRectFilled(
+                        { screen_pos.x + space_from_left + max_text_width + space_between_columns - i , line_y - i },
+                        { screen_pos.x + space_from_left + size.x + max_text_width + space_between_columns + i , line_y + size.y + i },
+                        ImColor( alpha( sensor_bg, 0.1f ) ) );
+                
+                ImGui::SetCursorScreenPos( { screen_pos.x + space_from_left + max_text_width + space_between_columns - 5.f, line_y - 3.f } );
```

Comment:
Please try to understand if we must set the Y again.
and why we need to manipulate it.
We need to understand the -5 and -3 , and avoid it if possible